libs/libs-pomfix/test/test.js

```diff
@@ -0,0 +1,57 @@
+var test = require('tap').test
```

Comment:
We probably need to add the usual copyright blurb at the top of all the `.js` files.